### ISSUE TITLE
[🐛 BUG] Installing Taipy GUI alone leads to error when Taipy runs

### ISSUE THREAD

**Issue by FlorianJacta** [Avaiga/taipy#1738]
### What went wrong? 🤔

When installing Taipy GUI alone, this leads to an error when importing taipy.

```
Traceback (most recent call last):
    ...
    from taipy._cli._base_cli._abstract_cli import _AbstractCLI
ModuleNotFoundError: No module named 'taipy._cli'
```

### Expected Behavior

Even when installing Taipy GUI alone, Taipy should run without issue.

### Steps to Reproduce Issue

1. Install `taipy-gui==4.0.0.dev1` alone (in a new environment)
2. Run a simple code `import taipy`

### Version of Taipy

4.0.0.dev1

### Acceptance Criteria

- [ ] Ensure new code is unit tested, and check code coverage is at least 90%.
- [ ] Create related issue in taipy-doc for documentation and Release Notes.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

**Comment by dinhlongviolin1** [comment]
I have looked into this issue, the problem is with the taipy-config package not including taipy._cli correctly. The same issue will happen if you install taipy-core independently. We will look into this one.

**Comment by dinhlongviolin1** [comment]
solved by @joaoandre in #1770
